### DISCUSSION TITLE
how to solve "Failed to initialize authentication subsystem" when run dockers

### DISCUSSION THREAD

**Discussion by lancepan** [microcks/microcks#1062] [Q&A] [answered]
I followed  the following tutorial  to run microcks. 
![image](https://github.com/microcks/microcks/assets/56334800/0c6545f3-4ba2-4a15-b6f2-b1e2b27dfb1e)

but when I open a new browser tab and point to the http://localhost:8080 endpoint, an error  always occured 
![image](https://github.com/microcks/microcks/assets/56334800/5d656874-9801-4712-9916-e7b1f2459c97)

What should I do to solve this issue?

**Comment by lbroudoux** [comment, marked as answer]
Hi @lancepan!

Thanks for trying out Microcks. Could you check you have all the containers running correctly after the docker-compose sdtartup?
You should have something like:

```shell
$ docker ps
CONTAINER ID   IMAGE                                              COMMAND                  CREATED              STATUS              PORTS                                                                NAMES
6f242d1dfcbe   quay.io/microcks/microcks:1.8.1                    "/deployments/run-ja…"   About a minute ago   Up About a minute   0.0.0.0:8080->8080/tcp, 8778/tcp, 0.0.0.0:9090->9090/tcp, 9779/tcp   microcks
4650f465e236   quay.io/microcks/microcks-postman-runtime:latest   "docker-entrypoint.s…"   About a minute ago   Up About a minute   3000/tcp                                                             microcks-postman-runtime
088357c6e439   mongo:3.6.23                                       "docker-entrypoint.s…"   About a minute ago   Up About a minute   27017/tcp                                                            microcks-db
0ce1e0945d54   quay.io/keycloak/keycloak:22.0.2                   "/opt/keycloak/bin/k…"   About a minute ago   Up About a minute   8443/tcp, 0.0.0.0:18080->8080/tcp                                    microcks-sso
```

Cna you also try to directly access `http://localhost:8180` in your browser? You should get access to the Keycloak instance there.

**Reply by lancepan** [reply]
Hi @lbroudoux  
Thanks for your answer.  There is indeed  a cotainner running incorrectly. Microcks runs well after I restart docker-compose. Thanks again.
